.github/workflows/e2e-test.yml

```
@@ -63,7 +63,7 @@ jobs:
       # Conditionally skip WC Blocks tests. Remove/update based on min supported WC version by the blocks checkout plugin.
       - name: Conditionally skip WC Blocks tests
         run: |
-          SKIP_WC_VERSIONS=('5.2.0' '6.1.0')
+          SKIP_WC_VERSIONS=('5.4.0' '6.5.1')
```

```suggestion
          SKIP_WC_VERSIONS=('5.4.0')
```

We only need to skip running WC Blocks tests against WC `5.4.0` since the minimum supported WC version for WC Blocks is `6.3`

---

Fixed.